### PR TITLE
CFM_183 update field validation to work with freedge fridge data import

### PR DESCRIPTION
**Pull request by 1jeanpaul1**
## Notes
Realized our field validations were too strict and made it so that certain fridges could not be added to our map. So updated the field validations, and updated the unit tests

## Updates
1. CommunityFridgeMapApi/dependencies/python/db.py
   * Updated Name and Id validation. The fridge name now accepts any characters and the ID is set to the fridge name with any non-ascii characters removed. 
   
2. README
    * Changed the order of commands users need to run
    
## Testing
1. Run the unit tests
2. Add a fridge using local API server or a one time request

### CODE REVIEW

File: CommunityFridgeMapApi/dependencies/python/db.py [hunk 1]
```diff
@@ -316,14 +317,15 @@ class Fridge(DB_Item):
...
+        "location/country": {"required": False, "max_length": 256},
         "maintainer": {"required": False, "type": "S"},
         "maintainer/name": {"required": False, "max_length": 256},
         "maintainer/organization": {"required": False, "max_length": 256},
         "maintainer/phone": {"required": False, "min_length": 10, "max_length": 20},
         "maintainer/email": {"required": False, "max_length": 320},
         "maintainer/website": {"required": False, "max_length": 2048},
         "maintainer/instagram": {"required": False, "max_length": 64},
-        "notes": {"required": False, "max_length": 280, "type": "S"},
+        "notes": {"required": False, "max_length": 700, "type": "S"},
```
Comment (1jeanpaul1) [inline CommunityFridgeMapApi/dependencies/python/db.py:328]: increased to 700, it's pretty long but didn't want to cut anyone short. There were only a few fridges that had really lengthy notes section. If we see that this causes the frontend to look too busy we can add a component on the frontend that blocks some of it and the user has to click on "see more" to see the rest of the notes perhaps?

File: CommunityFridgeMapApi/dependencies/python/db.py [hunk 2]
```diff
@@ -364,9 +366,6 @@ def __init__(self, db_client: "botocore.client.DynamoDB", fridge: dict = None):
             self.verified: bool = fridge.get("verified", None)
 
     def get_item(self, fridgeId):
-        is_valid, message = Fridge.is_valid_id(fridgeId=fridgeId)
```
Comment (1jeanpaul1) [inline CommunityFridgeMapApi/dependencies/python/db.py:367]: if the fridgeId is not valid then it won't be in the database anyways, so this validation is redundant

File: CommunityFridgeMapApi/dependencies/python/db.py [hunk 3]
```diff
@@ -435,35 +431,13 @@ def add_items(self):
...
 
-    def is_valid_name(self) -> bool:
-        """
-        Checks if a Fridge name is valid.
-            Returns:
-                bool (bool):
-        """
-        return re.match(r"^[A-Za-z0-9_#\-‘'.@& ]+$", self.name) is not None
-
-    def validate_fields(self) -> Field_Validator:
```
Comment (1jeanpaul1) [inline CommunityFridgeMapApi/dependencies/python/db.py:452]: uses the validate_fields function from parent class DB_ITEM instead

**Review by CollectiveFocus — APPROVED** [review]
LGTM

**Pull request merged**